scanner/src/main/kotlin/Main.kt

```diff
@@ -295,47 +280,43 @@ object Main {
             "To scan local files the chosen scanner must be a local scanner."
         }
 
-        println("Scanning path '${inputPath.absolutePath}'...")
-
-        val entry = try {
-            val result = (scanner as LocalScanner).scanPath(inputPath, outputDir)
+        val localScanner = scanner as LocalScanner
 
-            println("Detected licenses for path '${inputPath.absolutePath}': " +
-                    result.summary.licenses.joinToString())
+        println("Scanning path '${inputPath.absolutePath}'...")
 
-            SummaryEntry(
-                    detectedLicenses = result.summary.licenses,
-                    errors = result.summary.errors.toMutableList()
-            )
+        val result = try {
+            localScanner.scanPath(inputPath, outputDir).also {
+                println("Detected licenses for path '${inputPath.absolutePath}': ${it.summary.licenses.joinToString()}")
+            }
         } catch (e: ScanException) {
             e.showStackTrace()
 
             log.error { "Could not scan path '${inputPath.absolutePath}': ${e.message}" }
 
-            SummaryEntry(errors = e.collectMessages().toMutableList())
+            Instant.now().let {
```

Comment:
Nit: Maybe use `now ->` for readability here?

Reply:
Done. Also changed it to use `val now...` instead of `let` as you did in another PR.